projects/packages/my-jetpack/src/products/class-jetpack-ai.php

```diff
@@ -383,4 +531,61 @@ public static function get_ai_assistant_feature() {
 	private static function is_site_connected() {
 		return ( new Connection_Manager() )->is_connected();
 	}
+
+	/**
+	 * Get the URL where the user manages the product
+	 *
+	 * NOTE: this method is the only thing that resembles an initialization for the product.
+	 *
+	 * @return void
+	 */
+	public static function extend_plugin_action_links() {
+		add_action( 'admin_enqueue_scripts', array( static::class, 'admin_enqueue_scripts' ) );
+		add_filter( 'default_content', array( static::class, 'add_ai_block' ), 10, 2 );
+	}
+
+	/**
+	 * Enqueue the AI Assistant script
+	 *
+	 * The script is just a global variable used for the nonce, needed for the create post link.
+	 *
+	 * @return void
+	 */
+	public static function admin_enqueue_scripts() {
+		wp_register_script(
+			'my_jetpack_ai_app',
+			false,
+			array(),
+			Initializer::PACKAGE_VERSION,
+			array( 'in_footer' => true )
+		);
+		wp_localize_script(
+			'my_jetpack_ai_app',
+			'jetpackAi',
+			array(
+				'nonce' => wp_create_nonce( 'ai-assistant-content-nonce' ),
+			)
```

Comment:
would it make sense to pass this nonce to `myJetpackInititalState` instead?

Reply:
it might, but I usually prefer to leave things encapsulated on their own context. Adding a nonce to the initial state that only serves the AI product page seemed too specific for the overall (generic) purpose of My Jetpack's initial state. Open to suggestions though.